content/pt-br/docs/tasks/configmap-secret/_index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "Gerenciando Secrets"
+weight: 28
+description: Gerenciando dados de configurações confidencias usando Secrets.
```

Comment:
Although in original English it's using the "confidential" word, I'm a bit afraid that this wording makes users things that secrets and configmaps are secure and encrypted (which they are not, unless you configure them and just to be encrypted at rest).

Just some though.

Also, typo in confidencias / confidenciais

Reply:
I agree with that, maybe we can remove the 'Confidenciais' word and just use 'dados de configurações usando Secrets', what you think?

Reply:
@edsoncelio Use "Credenciais" instead "Secrets", what do you think?

Reply:
Hey @jcjesus thanks for the suggestion!  just to know, you're talking about the word 'Secrets' in the title or description?

Reply:
I think you can use both items.

Reply:
Nice, I agree with the first one, but to the second, the word 'Secret' it's used as a kubernetes resource name that will be use to create the credentials...

Reply:
Sure, I agree too! Thanks!